services/QuillLMS/db/migrate/20221209134742_remove_unit_from_pack_sequence_items.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class RemoveUnitFromPackSequenceItems < ActiveRecord::Migration[6.1]
```

Comment:
Note, you could consider collapsing all of the migrations in a new one now that you know the shape of the table(s). Fine to leave as is though, given we are trying to get this out.